docs/guides/external-node/docker-compose-examples/README.md

```diff
@@ -0,0 +1,36 @@
+# Running External node locally
+
+This directory contains two docker compose files for running external nodes, for sepolia testnet and mainnet.
+
+They are prepared to work out-of-the-box, no config changes are needed
+
+By default, the HTTP JSON-RPC API will run on port `3050`, while WS API will run on port `3051`.
+
+**Note: The node will recover from a snapshot on it's first run, this may take up to 10h.**
+
+**It needs to finish recovery, before the API server can serve any requests.**
+
+## Dependencies
+
+To run external node locally, you must have `docker compose` and `Docker` installed on your machine.
+
+## Usage
+
+To start testnet mainnet external node, run:
+
+```
```

Comment:
Nit: You can add `shell` language specifier here for syntax highlighting (and remove `>` below since it's not a valid shell syntax 🙃).

Reply:
corrected, thanks!